tests/wpunit/Tribe/Tickets/Partials/V2/TicketsTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tribe\Tickets\Partials\V2;
+
+use Codeception\TestCase\WPTestCase;
+use Spatie\Snapshots\MatchesSnapshots;
+use tad\WP\Snapshots\WPHtmlOutputDriver;
+
+class TicketsTest extends WPTestCase {
+
+	use MatchesSnapshots;
+
+	protected $partial_path = 'v2';
+
+	/**
+	 * @test
+	 */
+	public function test_should_not_render_if_not_is_sale_future__and__empty_provider_or_empty_tickets() {
```

Comment:
You should probably add testing variations where all three variants are covered (for instance, one test might be -- has provider, has is_sale_future, but has no tickets)